ush/python/pygfs/task/land_analysis.py

```diff
@@ -0,0 +1,265 @@
+        except Exception:
+            raise WorkflowException(f"An error occured during execution of {exec_cmd}")
+
+        pass
+
+    @logit(logger)
+    def finalize(self: Analysis) -> None:
+        """Finalize a global land analysis
+
+        This method will finalize a global land analysis using JEDI.
+        This includes:
+        - tarring up output diag files and place in ROTDIR
+        - copying the generated YAML file from initialize to the ROTDIR
+        - copying the guess files to the ROTDIR
+        - applying the increments to the original RESTART files
+        - moving the increment files to the ROTDIR
+
+        Please note that some of these steps are temporary and will be modified
+        once the model is able to read land increments.
+        """
+        # ---- tar up diags
+        # path of output tar statfile
+        landstat = os.path.join(self.task_config.COM_ATMOS_ANALYSIS, f"{self.task_config.APREFIX}landstat")
+
+        # get list of diag files to put in tarball
+        diags = glob.glob(os.path.join(self.task_config['DATA'], 'diags', 'diag*nc4'))
+
+        # gzip the files first
+        for diagfile in diags:
+            with open(diagfile, 'rb') as f_in, gzip.open(f"{diagfile}.gz", 'wb') as f_out:
+                f_out.writelines(f_in)
+
+        # open tar file for writing
+        with tarfile.open(landstat, "w") as archive:
+            for diagfile in diags:
+                diaggzip = f"{diagfile}.gz"
+                archive.add(diaggzip, arcname=os.path.basename(diaggzip))
+
+        # copy full YAML from executable to ROTDIR
+        src = os.path.join(self.task_config.DATA, f"{self.task_config.CDUMP}.t{self.runtime_config.cyc:02d}z.letkfoi.yaml")
+        dest = os.path.join(self.task_config.COM_ATMOS_ANALYSIS, f"{self.task_config.CDUMP}.t{self.runtime_config.cyc:02d}z.letkfoi.yaml")
+        yaml_copy = {
+            'mkdir': [self.task_config.COM_ATMOS_ANALYSIS],
+            'copy': [[src, dest]]
+        }
+        FileHandler(yaml_copy).sync()
+
+        # ---- NOTE below is 'temporary', eventually we will not be using FMS RESTART formatted files
+        # ---- all of the rest of this method will need to be changed but requires model and JEDI changes
+        # ---- copy RESTART sfc_data files for future reference
+        template = '{}.sfc_data.tile{}.nc'.format(to_fv3time(self.task_config.current_cycle), '{tilenum}')
+        bkglist = []
+        for itile in range(1, self.task_config.ntiles + 1):
+            sfcdata = template.format(tilenum=itile)
+            src = os.path.join(self.task_config.COM_ATMOS_RESTART_PREV, sfcdata)
+            dest = os.path.join(self.task_config.COM_ATMOS_ANALYSIS, f'landges.{sfcdata}')
+            bkglist.append([src, dest])
+        FileHandler({'copy': bkglist}).sync()
+
+        # ---- add increments to RESTART files
+        # logger.info('Adding increments to RESTART files')
+        # self._add_fms_cube_sphere_increments()
```

Comment:
This is commented out, then why is there a need for the method?

Reply:
The apply JEDI increments is implemented in the `scripts/exglobal_land_analysis_run.sh`. Therefore, `_add_fms_cube_sphere_increments` is not used here in the land DA and can be removed.

Reply:
Removed.